tests/trustyai/minio.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any
+
+from ocp_resources.pod import Pod
+from ocp_resources.secret import Secret
+from ocp_resources.service import Service
+
+
+MINIO: str = "minio"
+OPENDATAHUB_IO: str = "opendatahub.io"
+MINIO_DATA_CONNECTION_NAME: str = "aws-connection-minio-data-connection"
+
+
+class MinioPod(Pod):
+    def __init__(self, namespace: str, **kwargs: Any) -> None:
+        super().__init__(
+            name=MINIO,
```

Comment:
hard-coding the name of the resource would limit the use - only one in a namespace. does it make sense?

Reply:
Yes, as far as our use cases go we only need one minio data connection per namespace